.github/workflows/release.yml

```diff
@@ -24,15 +24,17 @@ jobs:
           fetch-depth: 0  # Fetch tags, which are required to calculate the new version
           token: "${{ secrets.GITHUB_TOKEN }}"
       - name: "Install commitizen"
-        run: pip install --user -U commitizen
+        run: |
+          pip install --user -U commitizen &&
+            cz version --verbose
       - id: tag
-        name: "Tag release"
+        name: "Release: Publish new semver Git Tag"
         run: |
-          VERSION=$(cz bump --get-next)
-          TAG="v${VERSION}"
-          git tag "${TAG}"
-          git push origin "${TAG}" --tags
-          echo "version=${VERSION}" >> "$GITHUB_OUTPUT"
+          NEW_VERSION=$(cz bump --get-next) || exit 1
+          TAG="v${NEW_VERSION}"
+          git tag "${TAG}" &&
+            git push origin "${TAG}" --tags &&
+            echo "version=${NEW_VERSION}" | tee "$GITHUB_OUTPUT"
```

Comment:
I see you switched to tee, but specifically away from an append to an overwrite. (The new version is equivalent to if you had switched from the `>>` you had to `>`, clobbering any prior state of the output file).

I don't _think_ this matters (idk what the GitHub docs say about that magic file) but if you want to append still then just use `-a` flag (`tee -a`)

Reply:
Ah good catch - all the github docs use append, I assume in-case you're populating multiple outputs which we're not doing here, but it's probably the safer choice